src/sliding_sync.rs

```diff
@@ -348,15 +357,24 @@ pub enum MatrixRequest {
         /// Whether to subscribe or unsubscribe from typing notices for this room.
         subscribe: bool,
     },
+    /// Subscribe to Updates for the given room.
+    ///
+    /// This request does not return a response or notify the UI thread.
+    SubscribeToUpdates {
```

Comment:
SubscribeToRoomUpdates
```suggestion
    SubscribeToRoomUpdates {
```